ci/homebrew/ci-deploy.sh

```diff
@@ -3,6 +3,7 @@
 set -e -o pipefail -u -x
 
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" < /dev/null
+(echo; echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"') >> /home/circleci/.bash_profile
```

Comment:
Hmmm what's the effect of **not** including this line? is it also sth related to 4.0.0 release of Homebrew? 🤔

Reply:
@PawelLipski No it's not, it's related to the lines 1418-1435 from https://app.circleci.com/pipelines/github/VirtusLab/git-machete/3148/workflows/cc6b52ba-d2e0-49cc-a0d2-bd41e383b4a9/jobs/16464

Reply:
Okay... pls include a comment in code that mentions that it's needed to avoid `Warning: /home/linuxbrew/.linuxbrew/bin is not in your PATH`